schema/deploy/shipping/views.sql

```diff
@@ -2198,6 +2199,96 @@ grant select
     to "scan-dashboard-exporter";
 
 
+create or replace view shipping.scan_demographics_v2 as
+
+    with hcov19_presence_absence as (
+        -- Collapse potentially multiple hCoV-19 results
+        select distinct on (sample_id)
+            sample.identifier as sample,
+            presence_absence_id,
+            case
+              when present is true then 'positive'
+              when present is false then 'negative'
+              when present is null then 'positive'
+            end as hcov19_result,
+            pa.created::date as hcov19_result_release_date
+        from
+            warehouse.presence_absence as pa
+            join warehouse.target using (target_id)
+            join warehouse.organism using (organism_id)
+            join warehouse.sample using (sample_id)
+        where
+            organism.lineage <@ 'Human_coronavirus.2019'
+            and pa.details @> '{"assay_type": "Clia"}'
+            and not control
+            -- We shouldn't be receiving these results from Samplify, but they
+            -- sometimes sneak in. Be sure to block them from this view so as
+            -- to not return inaccurate results to participants.
+            and target.identifier not in ('COVID-19_Orf1b', 'COVID-19-S_gene')
+        /*
+          Keep only the most recent push. According to Lea, samples are only
+          retested if there is a failed result. A positive, negative, or
+          indeterminate result would not be retested.
+
+          https://seattle-flu-study.slack.com/archives/CV1E2BC8N/p1584570226450500?thread_ts=1584569401.449800&cid=CV1E2BC8N
+        */
+        order by
+            sample_id, presence_absence_id desc
```

Comment:
In order to keep the most recent results push, I wonder if ordering by hcov19_result_release_date be more explicit than the id field

Reply:
There's an edge case of the same sample being tested twice in the same day, so ordering by release date would not work. See [previous discussions](https://github.com/seattleflu/id3c-customizations/pull/117#discussion_r454043119)